src/selectors/entities/users.js

```diff
@@ -192,6 +192,8 @@ function sortAndInjectProfiles(profiles, profileSet, skipInactive = false) {
         currentProfiles = Array.from(profileSet).map((p) => profiles[p]);
     }
 
+    currentProfiles = currentProfiles.filter((profile) => Boolean(profile));
+
```

Comment:
I'm not sure what the possible values of `profile` might be but be warned:

https://jsbin.com/kifinulaqa/edit?js,console

Reply:
Yep, this is just a filter on `falsy` check, specifically `undefined` in this case. 👍